template/src/index.template.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>{{ title }}</title>
+    <meta charset="utf-8">
```

Comment:
Research if we still need to add this, I think we do...
<meta http-equiv="X-UA-Compatible" content="IE=edge">